### PR TITLE
Introduce Quarg quantum hull repair

### PR DESCRIPTION
**Pull request by Nescio0**
Pug and Remnant ships have integrated hull repair and shield regenerators; Quarg ships have a separate shield generator, therefore it would make sense to have a separate hull repair as well. This proposal introduces a "Quantum Hull Repair", half the size of their shield generator, and two-thirds its cost, because hull repair is rarer than shield recharge. This proposal supersedes and reverts 9de051995dc7bb7f15a117460ee2c0243df483f2
I've also reordered the quantum shield generator near the other outfits, instead of behind the engines, and inserted a few white lines for consistency.
Because Quarg ships are practically uncapturable, Quarg outfits are not for sale, and Quarg outfits do not have sprites, this one doesn't need one either.

### CODE REVIEW

File: data/quarg outfits.txt [hunk 1]
```diff
@@ -26,6 +26,24 @@ outfit "Antimatter Core"
...
+	"shield generation" 10
+	"shield energy" 10
+
+outfit "Quantum Hull Repair"
+	category "Systems"
+	cost 20000000
+	thumbnail "outfit/unknown"
+	"mass" 60
+	"outfit space" -60
+	"shield generation" 5
```
Comment (tehhowch) [inline data/quarg outfits.txt]:
```suggestion
	"hull repair rate" 5
```
surely you didn't mean to make the hull gen shield gen

File: data/quarg outfits.txt [hunk 2]
```diff
@@ -26,6 +26,24 @@ outfit "Antimatter Core"
...
+	"shield energy" 5
```
Comment (tehhowch) [inline data/quarg outfits.txt]:
```suggestion
	"hull energy" 5
```

**Comment by Darcman99** [comment]
Shouldn't it be hull repair...?

**Comment by Nescio0** [comment]
Thank you for your quick replies! I've now corrected that shield typo.
This proposa can work in addition to and in combination with #3536. #3665, on the other hand, is more an overhaul that replaces the current Quarg content with similar named but different ships etc.

**Comment by Brick63** [comment]
#3665 already adds in a hull repair outfit

**Comment by KiLEdEnNis** [comment]
having various Quarg outfit & ship PR's open is a waste of time, since we have the one Brick listed
until that one's merged, it's better to list any idea on there to avoid a mess, and after that one's merged, people can look what to do next for the Quarg

**Comment by Nescio0** [comment]
Then I'd recommend reviewing and possibly merging #3536 first; it basically adds sprites to the existing Quarg outfits. I'm not really in favour of #3665; it tries to do too many things at once, not all of them are improvements.

Anyway, I've included @Brick63's hull repair sprite into this proposal.

File: copyright [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
```
Comment (jafdy) [inline copyright]: There seem to be quite a few unrelated changes in here, like this one.

Comment (tehhowch) [reply copyright]: Pretty sure this is another UTF-8 BOM getting chomped (or readded). A quick look at the format spec doesn't indicate that the BOM is required or disallowed.

Comment (Nescio0) [reply copyright]: Actually I have no idea what happened in this line.

Comment (Brick63) [reply copyright]: If you are using the desktop app, then you can deselect the top thing and it won't be in the update

Comment (Nescio0) [reply copyright]: No, I don't use desktop apps, I make edits directly in the browser. Anyone knows how to revert this line? (Or via a terminal command?)

Comment (tehhowch) [reply copyright]: Use something better than the browser. Then you can name your branches too

File: copyright [hunk 4]
```diff
@@ -269,11 +269,12 @@ Copyright: Zachary Siple
 License: CC-BY-SA-4.0
 
 Files:
- images/outfit/korath?rifle*
  images/outfit/hai?rifle*
- images/outfit/scram?drive*
  images/outfit/korath?fuel?processor*
+ images/outfit/korath?rifle*
+ images/outfit/quarg?hull?repair*
```
Comment (Amazinite) [inline copyright]: This outfit wasn't derived from works by MZ.

**Comment by Nescio0** [comment]
* #3536 has been committed as 87c22a9c5810fde83190b730f5310ad902dfd0e2
* a sprite in the same style has been added to this proposal
* copyright file has been updated (again)

**Comment by tehhowch** [comment]
The copyright line you add is already matched by the more general one with the same license (`images/outfit/quarg*` matches `images/outfit/quarg hull repair`)

**Comment by Nescio0** [comment]
You're right; reverted.

**Comment by Amazinite** [comment]
If you have an issue with #3665 then comment on it, but note that I'm keeping that PR on hold for a while, as it isn't a necessary change at the moment given that the Quarg aren't the focus of any major storyline (e.g. the Coalition storyline). 

I don't see a dire need for the Quarg to have an outfit for their hull repair at this current moment since hull repair is already baked in more for gameplay purposes than completion purposes. The Quarg are an underdeveloped faction right now, hence why #3665 seeks to expand upon them, but we don't need to be expanding upon them piece by piece, PR by PR that could conflict when #3665 will eventually do it all at once. (Think of it like adding a whole new race to the game where everything is added either at once or in large chunks. That's essentially what we're doing given how little the Quarg have right now.)
